alpaca_trade_api/stream.py

```diff
@@ -191,17 +193,23 @@ def _subscribe(self, handler, symbols, handlers):
             ).result()
 
     async def _subscribe_all(self):
-        if any(
-            v for k, v in self._handlers.items()
-            if k not in ("cancelErrors", "corrections")
-        ):
-            msg = {
-                k: tuple(v.keys())
-                for k, v in self._handlers.items()
-                if v
-            }
-            msg['action'] = 'subscribe'
-            await self._ws.send(msgpack.packb(msg))
+        symbolsInMsg = 0
+        msg = defaultdict(list)
+        for k, v in self._handlers.items():
+            if k not in ("cancelErrors", "corrections") and v:
+                for s in v.keys():
+                    if (symbolsInMsg == self._maxSymbolsInMessage):
+                        await self._send_subscribe_msg(msg)
+                        msg = defaultdict(list)
+                        symbolsInMsg = 0
+                    msg[k].append(s)
+                    symbolsInMsg += 1
+        if msg:
+            await self._send_subscribe_msg(msg)
+
+    async def _send_subscribe_msg(self, msg):
+        msg['action'] = 'subscribe'
+        await self._ws.send(msgpack.packb(msg))
```

Comment:
While it works, I think it's confusing to the users, because they will get back multiple subscription responses if the original subscription is broken down to multiple individual requests.

Can I suggest another solution, which sends the really big subscription message in smaller frames? Instead of sending multiple smaller subscription messages.
```python3
msg['action'] = 'subscribe'
bs = msgpack.packb(msg)
frames = ( bs[i:i+self._max_frame_size] for i in range(0, len(bs), self._max_frame_size) )
await self._ws.send(frames)
```

I got this from the [documentation](https://websockets.readthedocs.io/en/stable/reference/client.html#websockets.client.WebSocketClientProtocol.send): 
>[send()](https://websockets.readthedocs.io/en/stable/reference/client.html#websockets.client.WebSocketClientProtocol.send) also accepts an iterable or an asynchronous iterable of strings, bytestrings, or bytes-like objects to enable [fragmentation](https://www.rfc-editor.org/rfc/rfc6455.html#section-5.4). Each item is treated as a message fragment and sent in its own frame. All items must be of the same type, or else [send()](https://websockets.readthedocs.io/en/stable/reference/client.html#websockets.client.WebSocketClientProtocol.send) will raise a [TypeError](https://docs.python.org/3/library/exceptions.html#TypeError) and the connection will be closed.